wildfire_connector.py

```diff
@@ -448,7 +455,7 @@ def _normalize_children_into_list(self, input_dict):
         if not input_dict:
             return {}
 
-        for key in input_dict:
+        for key in list(input_dict):
```

Comment:
this change shouldn't be necessary:

`for key in list(input_dict)` == `for key in input_dict.keys()` == `for key in input_dict`

the only difference is the first one creates a copy of the keys before looping

Reply:
oh... right after I sent this I realized we're modifying the thing we're looping in the loop itself...

as a general practice it's best to avoid mutating things inside a loop over them. can we instead create a new dictionary before the loop instead and populate + return that as opposed to mutating `input_dict`?

Reply:
we should be good now.